src/constants/DefaultFields.ts

```diff
@@ -1,70 +1,90 @@
 import QuestionTypes from "./QuestionTypes";
 
+export enum DefaultFieldName {
+  ADDRESS = "address",
+  CELL_NUMBER = "cell_number",
+  EMAIL = "email",
+  FIRST_NAME = "first_name",
+  HOME_NUMBER = "home_number",
+  ID = "id",
+  LAST_NAME = "last_name",
+  NUM_CHILDREN = "num_children",
+  PHONE_NUMBER = "phone_number",
+  PREFERRED_CONTACT = "preferred_comms",
+  PREFERRED_NUMBER = "preferred_number",
+  WORK_NUMBER = "work_number",
+}
+
 export const DefaultFields = Object.freeze({
-  EMAIL: {
-    id: "email",
-    name: "Email",
+  ADDRESS: {
+    id: DefaultFieldName.ADDRESS,
+    name: "Address",
     question_type: QuestionTypes.TEXT,
   },
-  CELL_PHONE: {
-    id: "cell_number",
+  CELL_NUMBER: {
+    id: DefaultFieldName.CELL_NUMBER,
     name: "Cell Phone",
     question_type: QuestionTypes.TEXT,
   },
+  EMAIL: {
+    id: DefaultFieldName.EMAIL,
+    name: "Email",
+    question_type: QuestionTypes.TEXT,
+  },
   FIRST_NAME: {
-    id: "first_name",
+    id: DefaultFieldName.FIRST_NAME,
     name: "First name",
```

Comment:
nit: should then n be capitalized (same thing with `Last name`)? perhaps see designs

Reply:
oof good call, we actually use both sentence and proper case throughout the designs

will ask designers if this is intentional, but in the meantime updated them all to sentence case for consistency (we can add logic to convert them in certain places if necessary)